tasks/go.py

```diff
@@ -368,7 +368,11 @@ def check_go_mod_replaces(_ctx):
                     errors_found.add(f"{mod.import_path}/go.mod is missing a replace for {err_mod}")
 
     if errors_found:
-        message = "\nErrors found:\n" + "\n".join("  - " + error for error in sorted(errors_found))
+        message = "\nErrors found:\n"
+        message += "\n".join("  - " + error for error in sorted(errors_found))
+        message += (
+            "\n\nThis task operates on go.sum files, so make sure to run `inv -e tidy-all` before re-running this task."
```

Comment:
As this is mandatory we could maybe even run it as part of the task (and mention we've done it)

Reply:
This might get tricky since we run this in the CI now, I don't want the check to fix stuff (kind of how `check_mod_tidy` does not fix anything)
I'd like to work on a `fix-go-mod-replaces` at some point that would do that